dist/platforms/mac/steps/build.sh

```diff
@@ -130,6 +130,8 @@ echo ""
   -quit \
   -batchmode \
   -nographics \
+  -username "$UNITY_EMAIL" \
+  -password "$UNITY_PASSWORD" \
```

Comment:
These are already passed during the activate step. Any reason why we'd need to pass them here too?